<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uoDbListForm</class>
 <widget class="QWidget" name="uoDbListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Reference list</string>
  </property>
  <property name="windowIcon">
   <iconset resource="uodbmanager.qrc">
    <normaloff>:/uodb/images/table.png</normaloff>:/uodb/images/table.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="m_toolBar">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
     <addaction name="m_actNewElement"/>
     <addaction name="m_actNewGroup"/>
     <addaction name="m_actEdit"/>
     <addaction name="m_actView"/>
     <addaction name="m_actDelete"/>
     <addaction name="separator"/>
     <addaction name="m_actHierarhi"/>
     <addaction name="m_actRefresh"/>
     <addaction name="separator"/>
     <addaction name="m_actUpToParent"/>
     <addaction name="m_actDownToChildren"/>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_TitleLabel">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="m_searchCB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>114</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="uodbmanager.qrc">
         <normaloff>:/uodb/images/Searche.PNG</normaloff>:/uodb/images/Searche.PNG</iconset>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="m_splitter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>100</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="uoDbTreeView" name="m_treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="uoDbRefTableView" name="m_tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="m_ClosePB">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_actionPB">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Действия</string>
       </property>
       <property name="icon">
        <iconset resource="uodbmanager.qrc">
         <normaloff>:/uodb/images/table.png</normaloff>:/uodb/images/table.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="m_actNewElement">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/element_add.PNG</normaloff>:/uodb/images/element_add.PNG</iconset>
   </property>
   <property name="text">
    <string>Новый элемент</string>
   </property>
  </action>
  <action name="m_actNewGroup">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/folder_add.png</normaloff>:/uodb/images/folder_add.png</iconset>
   </property>
   <property name="text">
    <string>Новая группа</string>
   </property>
  </action>
  <action name="m_actEdit">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/element_edit.png</normaloff>:/uodb/images/element_edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="m_actView">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/element_review.png</normaloff>:/uodb/images/element_review.png</iconset>
   </property>
   <property name="text">
    <string>Просмотр</string>
   </property>
  </action>
  <action name="m_actDelete">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/element_g_delete.PNG</normaloff>:/uodb/images/element_g_delete.PNG</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="m_actHierarhi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/hierarhy.png</normaloff>:/uodb/images/hierarhy.png</iconset>
   </property>
   <property name="text">
    <string>Включить или выключить иерархию</string>
   </property>
  </action>
  <action name="m_actRefresh">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/refresh.png</normaloff>:/uodb/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить таблицу</string>
   </property>
  </action>
  <action name="m_actUpToParent">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/folder_down.png</normaloff>:/uodb/images/folder_down.png</iconset>
   </property>
   <property name="text">
    <string>Вверх к родителю</string>
   </property>
  </action>
  <action name="m_actDownToChildren">
   <property name="icon">
    <iconset resource="uodbmanager.qrc">
     <normaloff>:/uodb/images/folder_up.png</normaloff>:/uodb/images/folder_up.png</iconset>
   </property>
   <property name="text">
    <string>Вниз к элементу</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>uoDbRefTableView</class>
   <extends>QTableView</extends>
   <header>uodbreftableview.h</header>
  </customwidget>
  <customwidget>
   <class>uoDbTreeView</class>
   <extends>QTreeView</extends>
   <header>uodbtreeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tableView</tabstop>
  <tabstop>m_ClosePB</tabstop>
  <tabstop>m_actionPB</tabstop>
  <tabstop>m_searchCB</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>m_treeView</tabstop>
 </tabstops>
 <resources>
  <include location="uodbmanager.qrc"/>
 </resources>
 <connections/>
</ui>
